libraries/conf_helpers.rb

```diff
@@ -27,15 +36,15 @@ def self.read_config(conf_path)
     end
 
     def self.parse_config(conf_body)
-      current_stanza = 'default'
+      current_section = 'default'
       current_config = {}
 
       conf_body.each_line do |ln|
         case ln
         when /^\s*\[([^#]+)\]\s*/
-          current_stanza = Regexp.last_match[1]
+          current_section = Regexp.last_match[1]
         when /^\s*([^#]+?)\s*=\s*([^#]+?)\s*(?:#.*)?$/
```

Comment:
We probably need a more robust conf reader at some point... as I have heard that others are using line continuation characters in configs